### PR TITLE
grid_documentation_changes

### PR DESCRIPTION
**Pull request by rahuljhakant**
Changing the grid introduction line

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/getting_started/quick.en.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ that element. This is not only a time-saver but also an excellent way
 of learning Selenium script syntax.
 
 ## Grid
+Selenium Grid is a feature in Selenium that allows you to run test cases in different machines across different platforms. The control of triggering the test cases is on the local machine, and when the test cases are triggered, they are automatically executed by the remote machine
```
Comment (Harsha509) [inline docs_source_files/content/getting_started/quick.en.md]: Hi @rahuljhakant ,

Thanks for the PR. Can you please make sure to have a line width of 70-80

Thanks,
Harsha

Comment (Harsha509) [inline docs_source_files/content/getting_started/quick.en.md]:
```suggestion
Selenium Grid allows you to run test cases in different machines across different platforms. 
The control of triggering the test cases is on the local end, and when the test cases are triggered, they are automatically executed by the remote end.
```

File: docs_source_files/content/getting_started/quick.en.md [hunk 2]
```diff
@@ -28,7 +28,8 @@ that element. This is not only a time-saver but also an excellent way
 of learning Selenium script syntax.
 
 ## Grid
-Selenium Grid is a feature in Selenium that allows you to run test cases in different machines across different platforms. The control of triggering the test cases is on the local machine, and when the test cases are triggered, they are automatically executed by the remote machine
+Selenium Grid allows you to run test cases in different machines across different platforms. 
+The control of triggering the test cases is on the local end, and when the test cases are triggered, they are automatically executed by the remote end.
```
Comment (Harsha509) [inline docs_source_files/content/getting_started/quick.en.md]: Hi @rahuljhakant ,

Thanks for the PR. There are couple of changes required to get this pr merged.

1. Limit each line with max of 70-80 characters in line width. For example
this line 'The control of triggering the test cases is on the local end, and when the test cases are triggered, they are' contains 109 chars, you can spilt into two lines

2. The changes are done for only english version document. Still there are 7 translated pages (french, de, zh-ch, ko, nl etc) the same text changes should be done in those .md pages too.

Please let us know if you still need help.

Regards,
Harsha

Comment (rahuljhakant) [reply docs_source_files/content/getting_started/quick.en.md]: @Harsha509  Now I got your point. I will do the translations for all the languages and also Limit each line with a max of 70-80 characters in line width.

**Comment by Harsha509** [comment]
Hi @rahuljhakant,

Did you get a chance to look into the changes suggested by @diemol ?

Thanks,
Harsha.

**Comment by rahuljhakant** [comment]
Closing this PR as I'm facing the licensing issue.

**Pull request closed without merging**